integ/jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   roots: ['<rootDir>'],
-  testMatch: ['**/*.test.ts'],
+  testMatch: ['**/*.test.[jt]s'],
   transform: {
     '^.+\\.tsx?$': 'ts-jest'
   },
```

Comment:
Does this make both the `.ts` and `.js` tests run? It looks like this pull request is trying to standardize on running only the javascript files except in this one instance. Unless I'm missing something, we can probably restrict this to `**/*.test.js`.

Reply:
Since I moved away from packaging the tests I changed it back to just use ts.